src/common/utils.h

```diff
@@ -11,6 +11,9 @@
 namespace gfs {
 namespace common {
 
+// Constant values that are frequently referred to in the impls
+size_t const bytes_per_mb = 1024 * 1024;
```

Comment:
nit. better naming for constants is kBytesPerMb

Reply:
done